oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -1671,6 +1671,7 @@ void pldm::responder::oem_ibm_platform::Handler::updateContainerIDofProcLed()
 void pldm::responder::oem_ibm_platform::Handler::
     processPowerCycleOffSoftGraceful()
 {
+    std::cerr << "Received soft graceful power cycle request" << std::endl;
```

Comment:
Please remove Debug trace

Reply:
@sagisin i have requested archana to add traces ..as anything involving power offs e.t.c are critical operations and does not really happen much in field (so we dont really fill up the journal)